### PR TITLE
PE-2340: feat(plus button): adds the plus button for the NoDrives view

### PR DESCRIPTION
**Pull request by matibat**
Changelog

- Adds the Plus Button to the NoDrives view

--- Releases ---
iOS release: https://appdistribution.firebase.google.com/testerapps/1:305132849030:ios:06ea33a95a2e0d42ffce07/releases/506v4nbvevm8o

### CODE REVIEW

File: lib/pages/no_drives/no_drives_page.dart [hunk 1]
```diff
@@ -1,17 +1,64 @@
...
+          children: [
+            Center(
+              child: Text(
+                appLocalizationsOf(context).noDrives,
+                textAlign: TextAlign.center,
+                style: Theme.of(context).textTheme.headline6,
+              ),
+            ),
+            if (!kIsWeb)
+              BlocBuilder<DrivesCubit, DrivesState>(
```
[the remark this answers is not in the document]
Comment (matibat) [reply lib/pages/no_drives/no_drives_page.dart]: Created a new PlusButton widget
da80ff6cec275af76b7de6d708422c39b1279572

File: lib/components/plus_button.dart [hunk 2]
```diff
@@ -0,0 +1,49 @@
+import 'package:ardrive/components/new_button.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+import '../blocs/blocs.dart';
+
+/// Renders a Plus ("+") Button that opens the same menu than the New Button
+///
+/// **Note** In order for this Widget to be placed positioned absolute to the whole
```
Comment (matibat) [inline lib/components/plus_button.dart:10]:
```suggestion
/// **Note** In order for this Widget to be positioned absolute to the whole
```

**Review by thiagocarvalhodev — APPROVED** [review]
LGTM!

**Review by fedellen — APPROVED** [review]
👌

**Pull request merged**